src/code/templates/fv-egg-sort-game.js

```diff
@@ -76,8 +76,8 @@ let animationEvents = {
           animatingDrake = new BioLogica.Organism(BioLogica.Species.Drake, egg.alleles, egg.sex);
           targetBasketIndex = basketIndex;
 
-          const leftOffset = 400,
-                topOffset = 200;
+          const leftOffset = 375,
+                topOffset = 100;
```

Comment:
These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.